src/tests/system/tests/test_authentication.py

```diff
@@ -137,3 +137,58 @@ def test_authentication__using_the_users_email_address(client: Client, ad: AD, m
     assert client.auth.parametrize(method).password(
         "uSEr_3@alias-dOMain.com", "Secret123"
     ), "User uSEr_3@alias-dOMain.com failed login!"
+
+
+@pytest.mark.topology(KnownTopologyGroup.AnyAD)
+@pytest.mark.ticket(jira="RHEL-65848", gh=7690)
+@pytest.mark.parametrize("method", ["su", "ssh"])
+@pytest.mark.importance("high")
+def test_authentication__by_updating_id_provider_to_ldap(client: Client, provider: GenericADProvider, method: str):
+    """
+    :title: Login by updating id_provider to ldap
+    :description:
+        Testing the authentication of user when we update id_provider to ldap with AD.
+    :setup:
+        1. Add AD user.
+        2. Update the sssd.conf and Start SSSD
```

Comment:
Add the config parameters to setup, because this is what gets imported to create the test plan. 

Update sssd.conf with 'id_provider = ldap', 'ldap_schema = ad', 'ldap_id_use_start_tls = false', 'auth_provider = ad' and 'ldap_sasl_mech = gssspengo'